### ISSUE TITLE
Line separated array list is not parsed properly by ktmpl

### ISSUE THREAD

**Issue by suneeta-mall** [InQuicker/ktmpl#20]
In this template:
```
kind: "Template"
apiVersion: "v1"
metadata:
  name: "xx"
  annotations:
    description: "ss"
labels:
  template: "ss"
objects:
  - kind: Job
    apiVersion: batch/v1
    metadata:
      name: metadata-generation
      namespace: sp
    spec:
      template:
        metadata:
          name: somepod
        spec:
          containers:
          - name: metadata-generation-container
            image: some-image
            args: ["x", 
                  "generate"]       
parameters:
  - name: "REGION"
    description: "AWS region name"
    required: true
    parameterType: "string"
 ```

because of line sperated arg list, ktmpl fails to apply them variable and convert the yaml with variables. If I put the list in one line it works fine.

**Comment by jimmycuadra** [comment]
This may be an issue with the upstream YAML parser that ktmpl uses. I'll need to look into this a bit more to verify.

**Comment by suneeta-mall** [comment]
Sure. Just wanted to add that the list syntax - seems to work fine.
```
        spec:
          containers:
            - name: some-container
              image: blah blah
              args: 
                - "run"
                - "--cluster"
```

**Comment by jimmycuadra** [comment]
I think https://github.com/chyh1990/yaml-rust/pull/72 fixed this problem, but it hasn't been released yet. I'll leave this issue open until I'm able to use the new one in ktmpl and release a new version.

**Comment by dtolnay** [comment]
I believe this is fixed in yaml-rust 0.4.0.

**Comment by jimmycuadra** [comment]
This is fixed in the latest release: https://github.com/jimmycuadra/ktmpl/releases/tag/0.8.0

I don't have access to this repo anymore, so please close the issue if it's solved for you. Thanks!
